app/views/static/_home_cover.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="cover">
+  <div class="cover-image" style="background-image: url(<%= asset_path 'static/home-cover-04.jpg' %>)"></div>
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12 col-sm-7">
+        <div class="cover-text">
+          <h1 class="cover-text__top">Discover the greatest works of art ever painted.</h1>
+          <h2 class="cover-text__bottom">Supercharge your study session with curated lesson plans designed to help you learn things faster and remember them even longer.</h2>
+        </div>
+      </div>
+      <div class="visible-xs col-xs-12 signup-form">
+        <div class="form-group">
+          <button type="submit" class="btn btn-default btn-block">Signup for Free</button>
```

Comment:
I noticed that the form disappears when the screen is thin. http://recordit.co/5T1Mxs1IAx

Which is fine, but it does present a problem with the signup button. With the form present the signup button should submit the form. However with the form gone, the signup button should take the user to another page with a signup form.

The simpler solution would be to keep the form present on mobile. Up for discussion